errors/auth.js

```diff
@@ -1,10 +1,26 @@
+'use strict';
+const authTypes = require('../config/auth');
```

Comment:
@NicerHugs instead of require the `../config/auth` file directly could you update this to load from `../lib/config`. This is important because the authentication config settings can be overridden by the local.js file and/or environment specific config files. Loading in `../config/auth` directly will potentially overlook those overrides, whereas if you load in `../lib/config` those overrides will be applied.